### DISCUSSION TITLE
Export MODPATH tracks to VTK?

### DISCUSSION THREAD

**Discussion by BlakeLG** [modflowpy/flopy#1744] [Q&A] [answered]
Does the flopy.export.vtk module export MODPATH particle tracks? if so, how?

I've been looking through the flopy.export.vtk module documentation and example notebook. I followed steps and exported particles from my model as points viewable in Paraview, but not the tracks themselves. Am I missing something or is this not possible (yet)?

**Comment by jlarsen-usgs** [comment, marked as answer]
@BlakeLG 

We currently don't have support to export the particles as polyline tracks, however this could be a useful feature for further development. This discussion is linked in issue #1749

**Reply by wpbonelli** [reply]
@BlakeLG polylines are now included and should render in Paraview. See [this notebook](https://github.com/modflowpy/flopy/blob/develop/examples/Notebooks/flopy3_vtk_pathlines.ipynb) for an example. Note that array data (particle ID and time) now live on points of the exported VTK mesh, no longer on vertex cells.

To access this feature until the next release, flopy can be installed from the develop branch e.g. `pip install https://github.com/modflowpy/flopy/archive/develop.zip`

**Comment by BlakeLG** [comment]
Much appreciated! I will check it out
